tasks-core/src/commonMain/kotlin/net/opatry/tasks/data/TaskRepository.kt

```diff
@@ -223,6 +266,7 @@ class TaskRepository(
             }
             if (remoteTaskList != null) {
                 taskListDao.upsert(remoteTaskList.asTaskListEntity(localTaskList.id, localTaskList.sorting))
+                taskListIds[localTaskList.id] = remoteTaskList.id
```

Comment:
This isn't ideal for the next step as it would consider this local only list as remote from now on.
In particular, it will query its tasks which aren't there for sure…

This should be refactored another way.